.github/workflows/config-e2e.yml

```diff
@@ -0,0 +1,42 @@
+name: Firebase Remote Config Rollouts E2E Tests
```

Comment:
Nit: consider generalizing to just "Firebase Remote Config E2E Tests" -- though I think this is changeable later

Reply:
Sure. I included the Rollouts b/c I wasn't sure if we plan to use this in the future as an RC test app and the project has `Rollouts` in the name. But agreed we can change this later if we want.